### PR TITLE
Devcontainer README -> Clarification.

### PR DESCRIPTION
**Pull request by levalencia**
- Description: The contribution guidlelines using devcontainer refer to the main repo and not the forked repo.  We should create our changes in our own forked repo, not on langchain/main
  - Issue: Just documentation
  - Dependencies: N/A,
  - Tag maintainer: @baskaryan
  - Twitter handle: @levalencia

If no one reviews your PR within a few days, feel free to @-mention the same people again.

See contribution guidelines for more information on how to write/run tests, lint, etc: https://github.com/hwchase17/langchain/blob/master/.github/CONTRIBUTING.md
 -->

### CODE REVIEW

File: .devcontainer/README.md [hunk 1]
```diff
@@ -25,7 +29,7 @@ You can also follow these steps to open this repo in a container using the VS Co
 
 2. Open a locally cloned copy of the code:
 
-   - Clone this repository to your local filesystem.
+   - Fork and Clone this repository to your local filesystem.
```
Comment (baskaryan) [inline .devcontainer/README.md:32]: out of curiosity why do you need to fork for this?

Comment (levalencia) [reply .devcontainer/README.md:32]: According to the contribution guidelines we should not commit to the main branch directly but fork first instead and then create a pull request, right 

The devcontainer guidelines tell to open the original repo, so if you make changes in your container and then commit you are not following the guidelines right?

Or am I wrong?

Comment (baskaryan) [reply .devcontainer/README.md:32]: yea if you're looking to contribute you definitely need a fork. Thought some folks may just want to run langchain in a container, but guess it's safe to assume most folks may want to contribute as well 👍

**Comment by baskaryan** [comment]
thanks @levalencia

**Pull request merged**